### PR TITLE
custom claims

### PR DESCRIPTION
**Pull request by Firuz2006**

### CODE REVIEW

File: src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Persistence/ApplicationDbContext.cs [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+    ApplicationUserClaim, IdentityUserRole<Guid>, IdentityUserLogin<Guid>, IdentityRoleClaim<Guid>,
+    IdentityUserToken<Guid>>, IApplicationDbContext
 {
     public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) :
         base(options)
     {
+        UserClaims = Set<ApplicationUserClaim>();
+    }
+
+    public Task<int> SaveChangesAsync()
```
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Persistence/ApplicationDbContext.cs:19]: You can remove this

File: src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Persistence/ApplicationDbContext.cs [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 
-public sealed class ApplicationDbContext : IdentityDbContext<ApplicationUser, ApplicationRole, Guid>
+public sealed class ApplicationDbContext : IdentityDbContext<ApplicationUser, ApplicationRole, Guid,
+    ApplicationUserClaim, IdentityUserRole<Guid>, IdentityUserLogin<Guid>, IdentityRoleClaim<Guid>,
+    IdentityUserToken<Guid>>, IApplicationDbContext
 {
     public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) :
         base(options)
     {
+        UserClaims = Set<ApplicationUserClaim>();
```
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Persistence/ApplicationDbContext.cs:16]: You don't need to set this

File: src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Migrations/20230824090253_CustomClaims.cs
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Infrastructure/Migrations/20230824090253_CustomClaims.cs:1]: For now, remove migrations

File: src/MRA.Jobs.Identity/MRA.Identity.Domain/Entities/ApplicationUserClaim.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace MRA.Identity.Domain.Entities;
+
+public class ApplicationUserClaim:IdentityUserClaim<Guid>
+{
+    public string Slug { get; set; } = "";
```
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Domain/Entities/ApplicationUserClaim.cs:7]: You don't need to set an empty string

File: src/MRA.Jobs.Identity/MRA.Identity.Api/MRA.Identity.Api.csproj [hunk 4]
```diff
@@ -17,6 +17,10 @@
 
         <PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.2.2" />
         <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.3" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.10">
```
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Api/MRA.Identity.Api.csproj:20]: you can remove this

File: src/MRA.Jobs.Identity/MRA.Identity.Api/Controllers/AuthController.cs [hunk 5]
```diff
@@ -32,7 +32,7 @@ public async Task<IActionResult> Login([FromBody] LoginUserCommand request)
 
         if (result.Exception!=null)
         {
-            return BadRequest(result.Exception);
+            return BadRequest(result.Exception.ToString());
```
Comment (komdil) [inline src/MRA.Jobs.Identity/MRA.Identity.Api/Controllers/AuthController.cs:35]: ? Any reason?

**Review by komdil — CHANGES_REQUESTED** [review]

File: src/MRA.Jobs.Identity/MRA.Identity.Api/Controllers/AuthController.cs (hunk 5 above)
Comment (Firuz2006) [reply src/MRA.Jobs.Identity/MRA.Identity.Api/Controllers/AuthController.cs:35]: I should use logger?

**Review by komdil — APPROVED** [review]

**Review by komdil — APPROVED** [review]

**Review by Muhammadislom99 — APPROVED** [review]

**Pull request merged**